compiler/damlc/tests/src/DA/Test/ScriptService.hs

```diff
@@ -980,9 +980,23 @@ main =
                   , "  revoked === [CanActAs p1, CanReadAs p2]"
                   , "  rights <- listUserRights \"u1\""
                   , "  rights === []"
+                  , "testUserAlreadyExists = do"
+                  , "  u1 <- createUser (User \"u1\" None) []"
+                  , "  u2 <- createUser (User \"u1\" None) []"
+                  , "  pure ()"
+                  , "testUserNotFound = do"
+                  , "  u1 <- createUser (User \"u1\" None) []"
```

Comment:
maybe a bit simpler: drop the first line

Reply:
ok